ui/imports/utils/Utils.qml

```diff
@@ -983,4 +983,49 @@ QtObject {
         // #15331
         return true
     }
+
+    function toBase64(buffer) {
```

Comment:
Unfortunately I cannot use`Qt.btoa`, because it produces a wrong result (different from js `window.btoa`).

Reply:
I'm not sure why, but it just produces a different length result.
I've written some tests (in html and qml). And `toBase64` works similar to `btoa`.